app/code/AstoundDRudenko/Badge/Helper/Data.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+namespace AstoundDRudenko\Badge\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Serialize\SerializerInterface;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Simple helper for work with config values
+ *
+ * Class Data
+ * @package AstoundDRudenko\Badge\Helper
+ */
+class Data extends AbstractHelper
```

Comment:
Helpers (in a form we know them in M1) are not actual anymore in M2.
Please put the logic of your `AstoundDRudenko\Badge\Helper\Data` into a config model - 
`AstoundDRudenko\Badge\Model\Config`